science/memesuite

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                memesuite
+version             5.1.1
+categories          science biology
+platforms           darwin linux
+license             Noncommercial
+maintainers         uw.edu:cegrant uw.edu:wnoble gmail.com:tlawbailey
+description         Motif analysis toolkit
+long_description    MEME-Suite is a collection of tools for the analysis of \
+                    motifs in DNA and proteins.
+homepage            http://meme-suite.org
+master_sites        http://meme-suite.org/meme-software/macports \
+                    http://alternate.meme-suite.org/meme-software/macports
+distfiles           meme-5.1.1.tar.gz
```

Comment:
beautiful job for your first submission! This works -- as we usually do things, you could use ```meme-${version}``` like below. Also, if you use
```
distname    meme-${version}
```
the file extension will be automatically added, defaulting to ```.tar.gz``` but integrates nicely if you change to xz or some other system in future (you'd just have to put ```use_xz yes``` and everything would change over to the new extraction system, for example).

Reply:
Thanks for the encouragement! I've fixed the distname value as suggested